# afferenza del robot al punto di interesse pi√π vicino
string robot_name
string ipoint_name
float32 distance